app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
chưa áp dụng i18n vào đây à e

Reply:
Chỗ này em xin phép giữ nguyên ạ. Vì toàn là key word (Sample App là tên app) nên cho vào i18n em thấy chưa hợp lí ạ